src/components/login/LoginForm.vue

```diff
@@ -134,6 +134,23 @@ export default {
 };
 </script>
 
+<style lang="scss" scoped>
+button {
+  margin-right: 15px;
+  font-weight: 900;
+  background: #FFBA27;
+  padding: 10px 20px;
+  text-transform: uppercase;
+  font-size: 16px;
+  letter-spacing: 1px;
+  position: relative;
+
+  &:hover {
+    background: lighten(#FFBA27, 5%);
+  }
+}
+</style>
```

Comment:
You have this same CSS repeated 4 times (and I guess this will grow the more we use this component). Not a good practice. It's okay to components to have a default styling as long as you can override it. Or if you have 2 styles that you use often, you could pass the chosen one via props. There are many nice solutions that doesn't require repetition :)